modules/local/samtools/sort/main.nf

```diff
@@ -2,16 +2,17 @@ process SAMTOOLS_SORT {
     tag "$meta.id"
```

Comment:
I guess there is a reason to maintain a local version of samtools sort? Otherwise use the nf-core module